lxc/storage_volume.go

```diff
@@ -1807,7 +1807,12 @@ func (c *cmdStorageVolumeShow) Command() *cobra.Command {
     Will show the properties of a custom volume called "data" in the "default" pool.
 
```

Comment:
Can we add a statement here about what the valid types are that can be used?

Reply:
Added.